.github/workflows/tests-mpi.yml

```diff
@@ -90,10 +90,7 @@ jobs:
     - name: Tests
       run: |
         export OMPI_MCA_rmaps_base_oversubscribe=yes
-
-        if [ ${{ matrix.python-version }} != 3.11 ]; then
-          mpirun -n 2 -- pytest tests/integration_tests/test_pytorch_distributed.py
-        fi
+        mpirun -n 2 -- pytest tests/integration_tests/test_pytorch_distributed.py
```

Comment:
Could you separate this change to another PR?

Reply:
Done!